microcosm_flask/conventions/crud.py

```diff
@@ -52,9 +55,16 @@ def search(**path_data):
                 context = {}
                 items, count = return_value
 
-            # TODO: use the schema for encoding
-            response_data = PaginatedList(ns, page, items, count, definition.response_schema, **context).to_dict()
-            return make_response(response_data)
+            response_data = PaginatedList(
+                ns=ns,
+                page=page,
+                items=items,
+                count=count,
+                schema=definition.response_schema,
+                operation=Operation.Search,
+                **context
+            )
+            return dump_response_data(paginated_list_schema, response_data)
```

Comment:
No longer calling `to_dict()`; we end up building the response data the same as in other endpoints